tsconfig.build.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src"]
```

Comment:
Why do we need this?

Reply:
I did this because initially the root tsconfig.json was excluding tests which means my linter was complaining about decorators and code formatting was not being applied to the test folder.

So I added both folders to the root and then made the test folder be ignored for build so the artefacts wouldn't end up in the dist folder.